src/common/Avatar.js

```diff
@@ -28,7 +27,7 @@ class Avatar extends PureComponent {
     status?: UserStatus,
     realm: string,
     shape: 'square' | 'rounded' | 'circle',
-    onPress: () => void,
+    onPress?: () => void,
```

Comment:
Sure you will, and me as well because these are temporary fixes and atleast the CI will start working and it has been broken since a week!